packages/admin/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -29,13 +29,11 @@ const styles = () => {
 };
 
 function DefaultArrowUp(): React.ReactElement {
-    const { palette } = useTheme();
-    return <ChevronUp htmlColor={palette.primary.contrastText} />;
+    return <ChevronUp />;
 }
 
 function DefaultArrowDown(): React.ReactElement {
-    const { palette } = useTheme();
-    return <ChevronDown htmlColor={palette.primary.contrastText} />;
+    return <ChevronDown />;
 }
```

Comment:
You could now remove the DefaultArrowUp and DefaultArrowDown components altogether. However, I'm not sure if there wasn't a change to this in the theme refactor as well.

Reply:
Done